webpki-roots/Cargo.toml

```diff
@@ -22,3 +22,7 @@ webpki = { workspace = true }
 webpki-ccadb = { path = "../webpki-ccadb" }
 x509-parser = { workspace = true }
 yasna = { workspace = true }
+
+[features]
+default = []
+web = ["pki-types/web"]
```

Comment:
I think CI would benefit from an update that adds `--all-features` coverage.

Reply:
I see no point in defining an empty `default` feature.